cms/djangoapps/contentstore/management/commands/force_publish.py

```diff
@@ -0,0 +1,70 @@
+"""
+Script for force publishing a course
+"""
+from django.core.management.base import BaseCommand, CommandError
+from xmodule.modulestore import ModuleStoreEnum
+from xmodule.modulestore.django import modulestore
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+from .prompt import query_yes_no
+from .utils import get_course_versions
+
+# To run from command line: ./manage.py cms force_publish course-v1:org+course+run
+
+
+class Command(BaseCommand):
+    """Force publish a course"""
+    help = '''
+    Force publish a course. Takes one arguments:
+    <course_id>: the course id of the course you want to publish forcefully
+    '''
```

Comment:
We need to update the help, it now takes two arguments, course id and 'commit' when you're ready to commit it.

It works great (tested on a sandbox) and the help was the only thing I noticed.